File: WEEK5/bj6_14425.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');
let lineIndex = 0;

const [N, M] = input[lineIndex++].split(' ').map(Number);

const setS = new Set();
for (let i = 0; i < N; i++) {
    setS.add(input[lineIndex++]);
}

let count = 0;
for (let i = 0; i < M; i++) {
    if (setS.has(input[lineIndex++])) {
        count++;
    }
}

console.log(count);

File: WEEK5/bj6_14425.py
import sys

input = sys.stdin.readline

N, M = map(int, input().split())

setS = set()
for _ in range(N):
    setS.add(input().strip())

count = 0
for _ in range(M):
    if input().strip() in setS:
        count += 1

print(count)
